docs/platform-integration/essentials/troubleshooting.md

```diff
@@ -1,29 +1,32 @@
 ---
 title: "Troubleshooting"
-description: "Describes how to troubleshoot issues encountered when developing with the Xamarin.Essentials library."
+description: "Describes how to troubleshoot issues encountered when using .NET MAUI Essentials."
 ms.date: 01/06/2020
 no-loc: ["Microsoft.Maui", "Microsoft.Maui.Essentials"]
 ---
 
 # Troubleshooting
 
+<!-- TODO Will this still apply with essentials being merged into .NET MAUI? -->
```

Comment:
It could still apply. But I'd leave it out for now until we/anyone encounters this problem.